services/telemetry/telemetry.go

```diff
@@ -768,7 +768,7 @@ func (ts *TelemetryService) trackConfig() {
 		"is_default_global_relay_smtp_username": isDefault(*cfg.MessageExportSettings.GlobalRelaySettings.SmtpUsername, ""),
 		"is_default_global_relay_smtp_password": isDefault(*cfg.MessageExportSettings.GlobalRelaySettings.SmtpPassword, ""),
 		"is_default_global_relay_email_address": isDefault(*cfg.MessageExportSettings.GlobalRelaySettings.EmailAddress, ""),
-		"global_relay_smtp_server_timeout":      *cfg.EmailSettings.SMTPServerTimeout,
+		"global_relay_smtp_server_timeout":      *cfg.MessageExportSettings.GlobalRelaySettings.SMTPServerTimeout,
```

Comment:
I found this one with the tool :) We were using `cfg.EmailSettings.SMTPServerTimeout` for `GlobalRelayMessageExport` but now it has it's own value.

Reply:
Good catch :)